src/test/java/com/wikia/webdriver/testcases/mediatests/VetAddingVideoTests.java

```diff
@@ -25,25 +25,25 @@
 
   Credentials credentials = Configuration.getCredentials();
 
-  @Test(groups = {"VetAddVideo_001"})
+  @Test(groups = {"VetAddVideo_001"}, invocationCount = 20)
   @RelatedIssue(issueID = "QAART-889", comment = "test fails randomly")
   @Execute(asUser = User.USER)
   public void VetAddVideo_001_MessageWallProvider() {
     MessageWall wall = new MessageWall(driver).open(credentials.userName);
     String message = PageContent.MESSAGE_WALL_MESSAGE_PREFIX + wall.getTimeStamp();
     String title = PageContent.MESSAGE_WALL_TITLE_PREFIX + wall.getTimeStamp();
     MiniEditorComponentObject mini = wall.triggerMessageArea();
-    wall.clickBoldButton();
-    mini.switchAndWrite(message);
-    wall.setTitle(title);
-    VetAddVideoComponentObject vetAddingVideo = mini.clickAddVideo();
-    VetOptionsComponentObject vetOptions =
-        vetAddingVideo.addVideoByUrl(VideoContent.YOUTUBE_VIDEO_URL3);
-    vetOptions.setCaption(PageContent.CAPTION);
-    vetOptions.submit();
-    mini.verifyVideoMiniEditor();
-    wall.submit();
-    wall.verifyPostedMessageVideo(title);
+//    wall.clickBoldButton();
+//    mini.switchAndWrite(message);
+//    wall.setTitle(title);
+//    VetAddVideoComponentObject vetAddingVideo = mini.clickAddVideo();
+//    VetOptionsComponentObject vetOptions =
+//        vetAddingVideo.addVideoByUrl(VideoContent.YOUTUBE_VIDEO_URL3);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)